ceph-dev-build/config/definitions/ceph-dev-build.yml

```diff
@@ -1,6 +1,6 @@
 - job:
     name: ceph-dev-build
-    node: trusty || centos7 || xenial || master
+    node: huge_xenial_builder
```

Comment:
we will need to make this generic, like `executor` so that we can fire more of these light weight executor nodes